### ISSUE TITLE
Some amenity=motorcycle_parking are not rendered

### ISSUE THREAD

**Issue by Phyks** [gravitystorm/openstreetmap-carto#3407]
Hi,

### Expected behavior

I expect all the `amenity=motorcycle_parking` nodes to appear on the map.

### Actual behavior

I've just noticed that some `amenity=motorcycle_parking` are not rendered. This is the case for https://www.openstreetmap.org/node/5028869329#map=19/48.81712/2.31139 for example.

My wild guess is that this is coming from a conflict between `amenity=motorcycle_parking` and `bicycle_parking=stands`. This tagging is actually kind of expected (it would lack a `bike=designated` I think) as this parking is shared among motorcycles and bikes (same stands, no distinction between them).

### Links and screenshots illustrating the problem

https://www.openstreetmap.org/node/5028869329#map=19/48.81712/2.31139

![2018-09-21-090254](https://user-images.githubusercontent.com/3856586/45865247-268aea80-bd7d-11e8-8c4a-9730d47ee7e1.png)

Thanks!

**Comment by Tomasz-W** [comment]
Can you provide some more examples? The one above is propably a not showing "wontfix" because of road label. As I checekd this tag in Overpass turbo, everything seems to be ok both on nodes and areas.

**Comment by polarbearing** [comment]
It's rendered, e.g. https://www.openstreetmap.org/node/5248412211
As @Tomasz-W aid, it's just blocked by another icon.

**Comment by kocio-pl** [comment]
Thanks for reporting and for testing. It is unfortunate, but there's no way to always show all the elements on the map, so i will close this issue now. However feel free to share some other ideas in the future.

**Comment by Phyks** [comment]
Indeed, I missed the fact it was the street label which was hiding it. Thanks for the feedback!
